### PR TITLE
feat(web-analytics): Rename initial session properties from "Initial X" to "Session X"

### PR DESCRIPTION
**Pull request by robbie-c**
## Problem

Spot the difference (one is using person props, the other session pros)
image

Internal thread: https://posthog.slack.com/archives/C06R39WNY5B/p1714475094508139?thread_ts=1714474192.508889&cid=C06R39WNY5B

## Changes

Make the session props look like "Session UTM source".

I also considered changing the icon for person/session/etc props, but not everywhere where this text is used has an icon

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Screenshot:
Screenshot 2024-04-30 at 12 21 07

### CODE REVIEW

File: frontend/src/lib/taxonomy.tsx [hunk 1]
```diff
@@ -1096,7 +1096,7 @@ for (const [key, value] of Object.entries(CORE_FILTER_DEFINITIONS_BY_GROUP.event
     if (SESSION_INITIAL_PROPERTIES_ADAPTED_FROM_EVENTS.has(key)) {
         CORE_FILTER_DEFINITIONS_BY_GROUP.session_properties[`$initial_${key.replace(/^\$/, '')}`] = {
             ...value,
-            label: `Initial ${value.label}`,
+            label: `Session ${value.label}`,
```
Comment (neilkakkar) [inline frontend/src/lib/taxonomy.tsx:1099]: flyby to make sure searching will work if I just type "session" from now on, I don't think this works automatically

Comment (robbie-c) [reply frontend/src/lib/taxonomy.tsx:1099]: Ah true - I worked on this for some other properties! Will fix

File: frontend/src/lib/taxonomy.tsx [hunk 2]
```diff
@@ -1094,9 +1094,9 @@ for (const [key, value] of Object.entries(CORE_FILTER_DEFINITIONS_BY_GROUP.event
         CORE_FILTER_DEFINITIONS_BY_GROUP.person_properties[key] = value
     }
     if (SESSION_INITIAL_PROPERTIES_ADAPTED_FROM_EVENTS.has(key)) {
-        CORE_FILTER_DEFINITIONS_BY_GROUP.session_properties[`$initial_${key.replace(/^\$/, '')}`] = {
+        CORE_FILTER_DEFINITIONS_BY_GROUP.session_properties[`$session_${key.replace(/^\$/, '')}`] = {
```
Comment (Twixes) [inline frontend/src/lib/taxonomy.tsx:1097]: Doesn't this break the display of already-saved `$initial_`-prefixed filters?

**Review by Twixes — COMMENTED** [review]
Definitely makes sense UX-wise!

**Pull request merged**